crates/gossip/README.md

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
This is a good place to start, it shows the flow and renders properly if you "view file"